setup.py

```diff
@@ -0,0 +1,21 @@
+from setuptools import setup, find_packages
+
+setup(
+    name='nccs',
+    version='0.1',
+    description='NCCS Supply Chain module',
+    url='http://github.com/example',
```

Comment:
Maybe put here the actual url of this repository?

Reply:
Thanks 😁